fortio/README.md

```diff
@@ -0,0 +1,42 @@
+# Fortio on Istio
+
+Experimental istio on istio deployment:
+
+fortio.istio.io is running an istio deployment of the `fortio report` application
+
+Using envoy directly as the internet facing istio ingress.
+
+The data presented is pulled from a configurable google cloud storage or aws s3 bucket.
+
+## Initial setup:
+
+One time setup:
+
+- Istio:
+```
+# Istio 'perf' mode installation:
+sh -c 'sed -e "s/_debug//g" install/kubernetes/istio-auth.yaml | egrep -v -e "- (-v|\"2\")" | kubectl apply -f -'
+```
+
+- cert-manager
```

Comment:
Do these need to be separate steps if this is a one-time thing?

Reply:
adding an `all` target and some explanation as to why you may want to run just 1 step